src/structures/Message.js

```diff
@@ -565,6 +565,19 @@ class Message extends Base {
         : APIMessage.transformOptions(content, options, { reply: this.member || this.author }),
     );
   }
+  /**
+   * Quotes the message.
+   * @param {StringResolvable} [content=''] The content for the message
+   * @returns {Promise<Message|Message[]>}
+   * @example
+   * // Quote a message
+   * message.quote('Hey, I\'m a quote!')
+   *   .then(() => console.log(`Sent a quote to ${message.author.username}`))
+   *   .catch(console.error);
+   */
+  quote(content) {
+    return this.channel.send(`> ${this.content}\n${this.member || this.author} ${content}`);
```

Comment:
you should account for multi-line content

maybe also change the behaviour similar to that of [`Message#reply`](https://github.com/discordjs/discord.js/blob/stable/src/structures/Message.js#L561)

Reply:
> As discord themselves are planning to remove that feature in favor of proper reply (which may or may not get proper bot API support at some point) I don't personally see much value in this

This pull request is not required as it's probably going to get removed, so I will go a head and close this